recipe/meta.yaml

```diff
@@ -276,7 +276,7 @@ build:
 requirements:
   build:
     - 7za  # [win]
-    - python >=3.7
+    - python <3.10
```

Comment:
We could have kept both upper and lower limits like python >=3.7,<3.10.

Reply:
It pulls latest available so anyway wont pull lower python versions.